README.md

```diff
@@ -33,6 +33,7 @@ Feel free to raise PRs to enrich this repo with more questions. Also most of the
 |22  | [Implement a singleton class in Javascript.](#Implement-a-singleton-class-in-Javascript.) |
 |23  | [Lifecycle hooks of react with use of each hook?](#Lifecycle-hooks-of-react-with-use-of-each-hook?) |
 |24  | [Does react component only update on state change or also on prop change ?](#Does-react-component-only-update-on-state-change-or-also-on-prop-change-?) |
+|25  | [what is the future  of web components ?](#What-is-the-future-of-web-components-?) |
```

Comment:
@Sahilsen Questions that are open to debate wouldn't be the best ones to add here. Also, please add a placeholder for answer as well.

Reply:
Updated